src/main/java/uk/gov/hmcts/darts/event/service/impl/EventServiceImpl.java

```diff
@@ -19,11 +23,26 @@ public class EventServiceImpl implements EventService {
     private final EventMapper eventMapper;
     private final EventIdValidator eventIdValidator;
     private final EventRepository eventRepository;
+    private final DataAnonymisationService dataAnonymisationService;
 
     @Override
     public AdminGetEventForIdResponseResult adminGetEventById(Integer eventId) {
         eventIdValidator.validate(eventId);
         Optional<EventEntity> eventEntityOptional = eventRepository.findById(eventId);
         return eventMapper.mapToAdminGetEventsResponseForId(eventEntityOptional);
     }
+
+    @Override
+    @Transactional
+    public void adminObfuscateEveByIds(List<Integer> eveIds) {
+        eveIds.stream()
+            .map(this::getEventEntityById)
+            .distinct()
+            .forEach(dataAnonymisationService::anonymizeEvent);
+    }
```

Comment:
Why don't we directly expose a new method `anonymiseEvents` on the obfuscation service that takes the list of ids and have the controller call that direclty?
Seems a bit redundant to me going through the EventService